### PR TITLE
config from attributes

### PR DESCRIPTION
**Pull request by d10v**
Config via attributes, marshal to TOML.

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
 default['go']['packages']           	= ['bosun.org/cmd/scollector']
 
 default['scollector']['custom_repo'] = false
+default['scollector']['config'] = {
+  'Host' => 'http://127.0.0.1:8070',
+  'tags' => {
+    'environment' => node.chef_environment,
+    'role' => node.run_list.roles.first || 'unknown',
+  },
+  'ColDir' => node['scollector']['collectors_dir'],
```
Comment (zonorti) [inline attributes/default.rb:15]: CaMel CaSe?

File: providers/default.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  @config = new_resource.config
+  @run_context = run_context
+end
+
+action :create do
+  create_config
+end
+
+action :start do
+  #influxdb_service(:start)
```
Comment (zonorti) [inline providers/default.rb]: influxdb?

File: attributes/default.rb (hunk 1 above)
Comment (d10v) [inline attributes/default.rb:15]: Yeah, but that question is better suited for scollector repo :)

**Comment by zonorti** [comment]
:+1:

**Comment by vadzay** [comment]
:+1:

**Comment by melifaro** [comment]
:+1:

**Pull request merged**